src/estimators.jl

```diff
@@ -154,21 +191,22 @@ function LGBMRegression(;
     time_out = 120,
     machine_list_file = "",
     device_type="cpu",
+    force_col_wise = false,
+    force_row_wise = false,
 )
 
     return LGBMRegression(
         Booster(), "", objective, boosting, num_iterations, learning_rate, num_leaves,
         max_depth, tree_learner, num_threads, histogram_pool_size,
-        min_data_in_leaf, min_sum_hessian_in_leaf, lambda_l1, lambda_l2,
-        min_gain_to_split, feature_fraction, feature_fraction_seed,
-        bagging_fraction, bagging_freq, bagging_seed, early_stopping_round,
-        max_bin, data_random_seed, init_score,
-        is_sparse, save_binary, categorical_feature,
-        is_unbalance, drop_rate, max_drop, skip_drop, xgboost_dart_mode,
-        uniform_drop, drop_seed, top_rate, other_rate,
-        metric, metric_freq,
-        is_training_metric, ndcg_at, num_machines, local_listen_port, time_out,
-        machine_list_file, 1, device_type
+        min_data_in_leaf, min_sum_hessian_in_leaf, max_delta_step, lambda_l1, lambda_l2,
+        min_gain_to_split, feature_fraction, feature_fraction_bynode, feature_fraction_seed,
+        bagging_fraction, bagging_freq, bagging_seed, early_stopping_round, extra_trees,
+        extra_seed, max_bin, bin_construct_sample_cnt, data_random_seed, init_score,
+        is_sparse, save_binary, categorical_feature, use_missing,
+        is_unbalance, boost_from_average, drop_rate, max_drop, skip_drop,
+        xgboost_dart_mode,uniform_drop, drop_seed, top_rate, other_rate, min_data_per_group, max_cat_threshold,
+        cat_l2, cat_smooth, metric, metric_freq, is_training_metric, ndcg_at, num_machines, local_listen_port, time_out,
+        machine_list_file, 1, device_type, force_col_wise, force_row_wise,
```

Comment:
blep, whats the hardcoded 1 there? I feel like something isn't quite right here because it doesnt align with something which should be a `1`

Reply:
oh, no it seems ok